app/package.json

```diff
@@ -130,6 +131,7 @@
     "react": "^16.4.1",
     "react-dom": "^16.4.2",
     "react-stripe-elements": "^2.0.1",
+    "reset-css": "^3.0.0",
```

Comment:
are we sure this will only affect the consents stuff and not change any styles on the rest of manage (where they might be relying on browser defaults)?

Reply:
Yep I can't see how it'll affect the rest of the codebase, it's only imported by a module that's unique to `consents`.